Source/Directory.Build.props

```diff
@@ -7,7 +7,7 @@
 
   <!-- Boogie dependency -->
   <ItemGroup>
-    <PackageReference Include="Boogie.ExecutionEngine" Version="2.13.1" />
+    <PackageReference Include="Boogie.ExecutionEngine" Version="2.13.2" />
```

Comment:
```suggestion
    <PackageReference Include="Boogie.ExecutionEngine" Version="2.13.3" />
```

Reply:
On second thought, 2.13.3 includes another change that Dafny will need to be adapted for. I'll include that in my proof obligation description PR, because it fits naturally with functionality I'd like to include there anyway.